main/src/ca/uwaterloo/flix/language/phase/Eraser.scala

```diff
@@ -19,38 +19,40 @@ package ca.uwaterloo.flix.language.phase
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.ast.ErasedAst
 import ca.uwaterloo.flix.language.ast.ReducedAst
-import ca.uwaterloo.flix.language.phase.jvm.AnonClassInfo
-
-import scala.collection.mutable
 
 object Eraser {
 
   def run(root: ReducedAst.Root)(implicit flix: Flix): ErasedAst.Root = flix.phase("Eraser") {
 
-    //
-    // A mutable set to hold all type information about all closures in the AST.
-    //
-    implicit val ctx: Context = Context(mutable.Set.empty)
-
     val defs = root.defs.map { case (k, v) => k -> visitDef(v) }
     val enums = root.enums.map { case (k, v) => k -> visitEnum(v) }
 
-    ErasedAst.Root(defs, enums, root.entryPoint, root.sources, ctx.anonClasses.toSet)
+    val anonClasses = root.anonClasses.map {
```

Comment:
Would be prettier with a helper like with the `defs` and `enums`, but OK.

Reply:
This class will be deleted.